src/server/indices/service/indices_service.js

```diff
@@ -0,0 +1,20 @@
+import { search } from './lib';
+
+export class IndicesService {
+  constructor(callDataCluster) {
+    this._callDataCluster = callDataCluster;
```

Comment:
If you're going down the stateful route, I recommend calling this simply `_callCluster` or something. `_callDataCluster` implies that it is a function to call to the data cluster itself, whereas this could be any cluster.